### DISCUSSION TITLE
How build it for gnome 43?

### DISCUSSION THREAD

**Discussion by bushaev-denis** [wwmm/easyeffects#2698] [Q&A] [answered]
Hey, sorry if it will such corny, but how i can build it for gnome 43?
When i run
```
flatpak-builder build-dir --user --install util/flatpak/com.github.wwmm.easyeffects.Devel.json --force-clean --ccache
```
it returns error
```
Failed to init: Unable to find sdk org.gnome.Sdk version 44
```

(i cannot update gnome to 44 bc it freezed for debian now)

**Comment by wwmm** [comment]
I never tried to build the Flatpak package. Can it be built this way @vchernin ?

**Comment by vchernin** [comment, marked as answer]
Try 
```
flatpak-builder build-dir --install-deps-from=flathub --user --install util/flatpak/com.github.wwmm.easyeffects.Devel.json --force-clean --ccache
```

**Reply by bushaev-denis** [reply]
When  i run your command i got this.
```
Dependency Sdk: org.gnome.Sdk 44
Installing org.gnome.Sdk/x86_64/44 from flathub
error: No remote refs found for ‘flathub’
Error installing deps: running `flatpak --user install -y --noninteractive flathub org.gnome.Sdk/x86_64/44`: Child process exited with code 1
```

PS: i also tried to install `org.gnome.Sdk 44` with flatpak manually, and it was successfull, but still got this error.

**Reply by vchernin** [reply]
You need to add the flathub repo: 

```
flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
```

**Reply by bushaev-denis** [reply]
Of course I added it. After your message, I decided to check what problems there might be with this and found this option: `flatpak remote-add --user --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo`
It seems to have worked, something has started to install.

**Reply by vchernin** [reply]
That makes sense, flathub needs to be added to --user since flatpak-builder uses --user. I will add it to the wiki.

**Reply by bushaev-denis** [reply]
Thank you very much, it was builds successfully!
